app/build.gradle

```diff
@@ -37,6 +37,9 @@ android {
     androidExtensions {
         experimental = true
     }
+    buildFeatures {
+        viewBinding = true
+    }
 
     dataBinding.enabled = true
```

Comment:
remove the dataBinding line

Reply:
@rob729 removing this line is creating unexpected errors in the app. I have gone through many docs and articles, but nothing seems to remove those errors.

Reply:
On it.

Reply:
@rob729 removed the dataBinding line.
On removing this, errors were coming in the app. The main reason for those errors were that all the layouts files were wrapped in <layout> which was required for databinding, not for viewbinding.